Tests/MusicXMLTests/ScoreTests/ScoreTestSuite.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ScoreTestSuite.swift
+//  MusicXMLTests
+//
+//  Created by Ben Lu on 10/14/19.
+//
+
+import XCTest
+import MusicXML
+
+class ScoreTests: XCTestCase {
+
+    func testAll() throws {
+        let paths = try FileManager.default.contentsOfDirectory(atPath: testSuiteURL.path)
+        for path in paths {
+            let testCaseURL = testSuiteURL.appendingPathComponent(path)
+            guard directoryExists(at: testSuiteURL.path) else {
+                continue
+            }
+            let manifestURL = testSuiteURL.appendingPathComponent(path).appendingPathComponent("manifest.plist")
+            guard let manifestData = FileManager.default.contents(atPath: manifestURL.path) else {
+                continue
+            }
+            let manifest = try PropertyListDecoder().decode(ScoreTestCaseManifest.self, from: manifestData)
+            try runTest(forManifest: manifest, testCaseURL: testCaseURL)
+        }
+    }
+
+    private func runTest(forManifest manifest: ScoreTestCaseManifest, testCaseURL: URL) throws {
+        let musicXMLURL = testCaseURL.appendingPathComponent(manifest.xmlFileName)
+
+        print("🎼 Test score: \(manifest.displayName)")
+        print("  - Path: \(musicXMLURL.path)")
+        if let description = manifest.description {
```

Comment:
Depending on your FP orthodoxy, this may be an abuse of `map`, but I think I like it a bit better:

```Swift
manifest.description.map { print("  - Description: \(description)") }
```